opslevel/datasource_opslevel_service.go

```diff
@@ -52,7 +52,7 @@ func datasourceService() *schema.Resource {
 				Description: "The software tier that the service belongs to.",
 				Computed:    true,
 			},
-			"owner_alias": {
```

Comment:
you need to keep the original field around but add the `Deprecated: true` to it and ensure it stays filled with data